optuna/dashboard.py

```diff
@@ -255,8 +255,8 @@ def _get_this_source_path():
     return path
 
 
-def serve(study):
-    # type: (optuna.study.Study) -> None
+def serve(study, bokeh_allow_websocket_origins=None):
+    # type: (optuna.study.Study, List[str]) -> None
```

Comment:
Thank you for changing the default argument value.
Now that `bokeh_allow_websocket_origins` accepts `None`, we should also change the type hint from `List[str]` to `Optional[List[str]]`.
Not all optional arguments require `Optional`, but this case is corresponding to the second example in [the reference of typing](https://docs.python.org/3.6/library/typing.html#typing.Optional), and I recommend adding `Optional`.